src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -263,6 +342,16 @@
  *   &lt;property name=&quot;allowEmptyLoops&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public static void main(String[] args) {
+ *         for (int i = 100;i &gt; 10; i--){} // ok
+ *         do {} while (i = 1); // ok
```

Comment:
example should always contain code with violations and without.
to make it clear that Check is still working but some cases become allowed.

please apply the same for other code samples.

Reply:
How can i show a violation here as the config says to allowEmpltyLoops?

Reply:
Done @romani